lib/carbon/conf.py

```diff
@@ -115,6 +115,7 @@
   TCP_KEEPINTVL=30,
   TCP_KEEPCNT=2,
   USE_RATIO_RESET=False,
+  LOG_LISTENER_CONN_LOST=True,
```

Comment:
I think we should keep compatibility with old versions. Could you please change default to False?

Reply:
I've updated this value to "False".